app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is a class variable, meaning it is shared across all instances of the `Person` class. This can lead to issues if you have multiple instances of `Person` with the same name, as they will overwrite each other in this dictionary. Consider using a different approach if unique identification is required.